torch_xla/csrc/tensor.h

```diff
@@ -205,6 +205,17 @@ class XLATensor {
   static XLATensor nll_loss_backward(const XLATensor& input,
                                      const XLATensor& target);
 
+  static std::tuple<XLATensor, XLATensor, XLATensor> native_batch_norm(
```

Comment:
What are the differences among the native and not?

Reply:
It's the forward version used for autograd, I've added a comment here as well (there's one for `NativeBatchNormForward`).